controllers/kustomization_decryptor_test.go

```diff
@@ -877,13 +882,13 @@ func TestKustomizeDecryptor_decryptKustomizationEnvSources(t *testing.T) {
 					GeneratorArgs: kustypes.GeneratorArgs{
 						Name: "envSecret",
 						KvPairSources: kustypes.KvPairSources{
-							FileSources: []string{"file.txt"},
-							EnvSources:  []string{"app.env", "key=../secret.env"},
```

Comment:
Note that _kustomize_ doesn't split the "EnvSources" field's values on an equals sign; it assumes that the whole value is always a file path.